src/components/mapgl/mapgl.component.css

```diff
@@ -46,37 +46,84 @@
   box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14),
     0 1px 5px 0 rgba(0, 0, 0, 0.12);
 }
-
-.map__visu-list--container /* @doc Sets the style of layers switcher container*/
- {
+.map__visu-list--wrapper {
   position: absolute;
   display: flex;
   flex-direction: column;
-  top: 250px;
-  right: 10px;
+  bottom: 165px;
+  right: 8px;
+  background: transparent;
+
+}
+.map_legend-button--wrapper {
+}
+.map__visu-list--container /* @doc Sets the style of layers switcher container*/
+ {
   background: white;
-  padding: 5px;
+  padding: 1px;
   box-shadow: 0 3px 1px -2px rgba(0, 0, 0, 0.2), 0 2px 2px 0 rgba(0, 0, 0, 0.14),
     0 1px 5px 0 rgba(0, 0, 0, 0.12);
   border-radius: 2px;
+  max-height: 590px;
```

Comment:
This height is relevant on lot of resolutions ?

Reply:
`bien vu`, it's not, I'll fix it

Reply:
fixed